ci/build_utils.sh

```diff
@@ -0,0 +1,42 @@
+function build_xen() {
+    # Expects the cwd to be Xen repository
+    PREFIX=$1
+
+    ./configure --prefix=$PREFIX --enable-githttp --disable-pvshim > /dev/null 2>&1
+    make -j$(nproc) dist > /dev/null 2>&1
+    make -j$(nproc) install-xen
+    make -j$(nproc) install-tools
+}
+
+function build_libvmi() {
+    # Expects the cwd to be libvmi repository
+    PREFIX=$1
+
+    mkdir -p $(pwd)/build
+    cd build
+    cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX \
+             -DENABLE_FILE=OFF \
+             -DENABLE_LINUX=OFF \
+             -DENABLE_FREEBSD=OFF \
+             -DENABLE_KVM=OFF \
+             -DENABLE_BAREFLANK=OFF
+    make -j$(nproc)
+    make install
```

Comment:
Looks like we are missing `--enable-vmi-debug` (is there any `cmake` equivalent?)

Reply:
:thinking: we are also missing `--disable-safety-checks` but I don't think this should be included. Be aware that some bugs might appear that we were recently using the package compiled with `--disable-safety-checks` o_O

Reply:
added the `-DENV_DEBUG=ON` flag